source/geometries/FullRingInfinity.cc

```diff
@@ -422,6 +464,117 @@ void FullRingInfinity::BuildSensors()
   }
 }
 
+void FullRingInfinity::BuildWires()
+{
+  // Add simple detector for charge
+  G4Box* chdet_solid = new G4Box("WIRE", wire_pitch_/2.,
+                                 axial_length_/2., chdet_thickn_/2);
```

Comment:
Because this way they intercept (almost) all the ionization electrons that come towards them. The idea is that they have the same shape as SiPMs, so that they can be placed exactly in front of them.